compiler/src/dotty/tools/dotc/core/JavaNullInterop.scala

```diff
@@ -77,12 +77,12 @@ object JavaNullInterop {
    *  If tp is a type of a field, the inside of the type is nullified,
    *  but the result type is not nullable.
    */
-  private def nullifyExceptReturnType(tp: Type)(using Context): Type =
-    new JavaNullMap(true)(tp)
+  private def nullifyExceptReturnType(tp: Type, ownerIsClass: Boolean)(using Context): Type =
```

Comment:
I don't think we need to modify `nullifyExceptReturnType`? It is a separate feature.

Reply:
Did you mean it's for unsafe-java-return? It looks like that's not what it's used for. See the call to `nullifyExceptReturnType` above.